static/js/components/UserProfileInfo.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+
+import UserAvatar from './UserAvatar';
+
+const UserProfileInfo = () => {
+  const profile = useSelector((state) => state.profile);
+
+  return (
+    <Card>
+      <CardContent>
+        <div style={{ display: "flex",
+          justifyContent: "flex-start",
+          flexDirection: "row",
+          alignItems: "center" }}
+        >
+          <UserAvatar
+            size={128}
+            firstName={profile.first_name}
+            lastName={profile.last_name}
+            username={profile.username}
+            gravatarUrl={profile.gravatar_url}
+          />
+            &nbsp;&nbsp;
```

Comment:
Instead of these, we can use https://material-ui.com/system/spacing/, in this case probably `mr`.

Reply:
will hold off on this - should make this apart of the styles revamp